src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_AutoSuggestBox.cs

```diff
@@ -62,8 +65,20 @@ public async Task When_Text_Changed_And_Focused_Should_Open_Suggestion_List()
 		}
 
 		[TestMethod]
+#if __SKIA__
+		[DataRow(true)]
+		[DataRow(false)]
+		public async Task When_Text_Changed_UserInput(bool useTextBoxOverlay)
+#else
 		public async Task When_Text_Changed_UserInput()
+#endif
 		{
+#if __SKIA__
+			var oldUseOverlay = FeatureConfiguration.TextBox.UseOverlayOnSkia;
+			FeatureConfiguration.TextBox.UseOverlayOnSkia = useTextBoxOverlay;
+			using var _ = Disposable.Create(() => FeatureConfiguration.TextBox.UseOverlayOnSkia = false);
```

Comment:
I believe you wanted to use the old value here
```suggestion
			using var _ = Disposable.Create(() => FeatureConfiguration.TextBox.UseOverlayOnSkia = oldUseOverlay);
```

Reply:
Yup, my bad.